### DISCUSSION TITLE
Export as HTML

### DISCUSSION THREAD

**Discussion by dishant26** [mindee/doctr#1005] [Q&A]
This functionality is shown in the README file. 
```
from doctr.io import DocumentFile
from doctr.models import ocr_predictor

model = ocr_predictor(pretrained=True)
# PDF
doc = DocumentFile.from_pdf("path/to/your/doc.pdf")
# Analyze
result = model(doc)
```
In the README it is shown that we can export this as json using `json_output = result.export()`
Can we export the `result` as HTML?

**Comment by frgfm** [comment]
Hello @dishant26 :wave:

I'll try my best to help! Here is some relevant information:
- the snippet that you show is about **reading** PDF, not exporting to PDF. Likewise, we can read from HTML content :)
- regarding the export, you can export as a JSON file, as an XML file (which you can use to produce PDF), or synthesize a white page with the detected text. You can use the first two to produce HTML content but this is not natively supported :sweat_smile: 

If I may, what is the exact use case behind this?
Cheers :v:
